src/components/schemas/statements.yaml

```diff
@@ -0,0 +1,151 @@
+      type: object
+      properties:
+        description_column:
+          type: string
+          description: |
+            Specifies the header name of the description column in the statement.
+          nullable: true
+          example: description
+        transaction_date_column:
+          type: string
+          description: |
+            Specifies the mapping of transaction date on the statement.
+          example: txn date
+        transaction_id_column:
+          type: string
+          description: |
+            Specifies the mapping of transaction id / unique id on the statement.
+          example: reference number
+          nullable: true
+        amount_column:
+          type: string
+          description: |
+            Specifies the mapping of amount on the statement.
+          example: amount
+        card_number_column:
+          type: string
+          description: |
+            Specifies the mapping of card number on the statement.
+          example: account number
+        vendor_column:
+          type: string
+          description: |
+            Specifies the vendor of the statement
+          example: Amazon
+        foreign_amount_column:
+          type: string
+          description: |
+            Specifies the mapping of foreign amount on the statement.
+          example: foreign amount
+          nullable: true
+        foreign_currency_column:
+          type: string
+          description: |
+            Specifies the mapping of foreign currency on the statement.
+          example: foreign currency
+          nullable: true
+        bank_name:
+          type: string
+          description: |
+            Specifies the bank name of the statement
+          example: American Express
+        is_debit_positive:
+          type: boolean
+          description: |
+            Specifies the representation of the the amount in the statement, if true then the amount represented by |
+            negative sign are credits, and if false then the amount represented by negative sign are debits.
+          example: true
+        currency:
+          type: string
+          description: |
+            Specifies the currency used in the statement.
+          example: USD
+        name:
+          type: string
+          description: |
+            Specifies the mapping name.
+          example: USD
+          nullable: true
+    ignored_line_items:
+      type: array
+      items: 
+        type: string
+      description: >
+        This specifies list of line items to be ignored in the statement.
+      example:
+        - stlisfdfadssd
+  required:
+    - id
+    - statement_mapping
+    - ignored_line_items
+
+preview_out:
+  properties:
+    transactions:
+      type: array
+      items: 
+        type: object
+        properties:
+          description:
+            type: string
+            description: |
+              Specifies the header name of the description column in the statement.
```

Comment:
This is not the header name @Kulsekar4 

Can use something like below with some changes.
`Description of the transaction in the statement as per description column mapping.`